schemas/common/context.jsonld

```diff
@@ -28,6 +28,7 @@
     "activitystreams": "https://www.w3.org/ns/activitystreams",
     "schema": "http://schema.org",
     "repo": "http://ns.adobe.com/adobecloud/core/1.0/",
-    "iptc4xmpExt": "http://iptc.org/std/Iptc4xmpExt/1.1/"
+    "iptc4xmpExt": "http://iptc.org/std/Iptc4xmpExt/1.1/",
+    "search" : "https://ns.adobe.com/search"
```

Comment:
There is no reason to create a new namespace, and it is to be avoided. These are additions to the Adobe model which all go under `xdm`. The only cases we use other namespaces is because we want to include third-party or preexisting historical standards.

Reply:
Actually, @cdegroot-adobe , @nandaja is absolutely doing the right thing (at my request).  We *do* want multiple namespaces to properly segment things.

Reply:
@lrosenthol - very interesting, this is interesting to know, I think we will want to consider several aspects of this for the XC area too, especially around identity.